Geometry/TrackerCommonData/data/cms-test-ddpixbarlayerupgrade-algorithm.xml

```diff
@@ -17,15 +17,15 @@
   <close_geometry/>
 
   <IncludeSection>
-    <Include ref="Geometry/TrackerCommonData/data/testDDPixBarLayerUpgradeAlgorithm.xml"/>
-    <Include ref="Geometry/CMSCommonData/data/materials/2015/v1/materials.xml"/>
-    <Include ref="Geometry/TrackerCommonData/data/trackermaterial.xml"/>
+    <Include ref="Geometry/CMSCommonData/data/materials/2021/v1/materials.xml"/>
```

Comment:
In this test data file, `v1` of `materials.xml` needed to be used instead of the more recent `v3`.